### DISCUSSION TITLE
Ever since I upgraded, I've encountered a lot of bugs.(1.2.5)

### DISCUSSION THREAD

**Discussion by iRexLyys** [Chia-Network/chia-blockchain#8271] [Support]
I just see the error you see in the picture below, but I am encountering different errors and I cannot actively dig in the pool right now. If the plots pass through the filter normally for 30 minutes, then the synchronization process is constantly renewing itself, the filtering process freezes and I have to press the "refresh" button all the time. How can I fix this problem? I've been trying for 3 days, but I couldn't find a solution. Problems started when the 1.2.5 version was released. There was no problem in the 1.2.3 version, so the 1.2.5 version was shared. Such problems started to occur. Will there be a new version or a fix version? Or are we going to fix it ourselves by doing something?

Sometimes there is a problem like this, an error message saying "1","2","3" appears on the screen out of the blue.

2021-08-29T22:38:20.172 wallet chia.rpc.rpc_server        : WARNING  Error while handling message: Traceback (most recent call last):
2021-08-29T22:53:29.834 harvester chia.harvester.harvester: ERROR    Error using prover object Src size is incorrect

Can we fix these errors? I will read your help in the comments.

![image](https://user-images.githubusercontent.com/89747893/131263296-9b14be54-d5f3-4845-962a-57afadc10042.png)

**Comment by loppefaaret** [comment]
`harvester chia.harvester.harvester: ERROR Error using prover object Src size is incorrect` is a problem for the harvester accessing a plot - it can be from a malformed plot, to a dying drive, bad memory, normally, that error lies outside of chia itself - perhaps try and diagnose that your hardware is working correctly is my best bet - im not saying that it is your hardware 100% - just that it is most of the times the place when you have issues like this, and is perhaps a place to start

**Reply by iRexLyys** [reply]
It's not a big deal to have that problem, I fix it, but what I want to point out is that the files that pass through the filter in the "Farm" section freeze and do not activate until I press the "Refresh" button. originating?"

As it can be seen, there is a constant decrease in the pool, it was progressing in a stable manner until the last update was shared.
When the last Proof-of-Try partition is frozen, it doesn't filter files at all.

![image](https://user-images.githubusercontent.com/89747893/131347809-8113ddfc-3ecf-4273-a396-afb7bac56be5.png)
![image](https://user-images.githubusercontent.com/89747893/131347868-951fea58-cd90-4f41-93cc-5603da9af44a.png)

**Comment by BrandtH22** [comment]
@iRexLyys , the newest version(s) of chia has fixes in place to resolve this issue. Please download it from here: https://www.chia.net/downloads/

Since this issue has been resolved with the latest version(s) of chia we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
